tests/retryable-writes/retryable-writes-001.phpt

```diff
@@ -32,7 +32,7 @@ class TransactionIdObserver implements MongoDB\Driver\Monitoring\CommandSubscrib
 $observer = new TransactionIdObserver;
 MongoDB\Driver\Monitoring\addSubscriber($observer);
 
-$manager = new MongoDB\Driver\Manager(URI, ['retryWrites' => true]);
+$manager = new MongoDB\Driver\Manager(URI);
```

Comment:
Looks like I missed these in https://github.com/mongodb/mongo-php-driver/pull/996. Good catch.